kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java

```diff
@@ -544,20 +549,20 @@ private static boolean isLastArgumentOfSuspendedMethod(Argument<?> argument, Exe
 
     private void configureDeserializers(final ExecutableMethod<?, ?> method, final DefaultKafkaConsumerConfiguration<?, ?> config) {
         final boolean batch = method.isTrue(KafkaListener.class, "batch");
-        final Argument<?> bodyArgument = findBodyArgument(batch, method);
+        final Optional<Argument<?>> bodyArgument = findBodyArgument(batch, method);
         configureKeyDeserializer(bodyArgument, method, config);
         configureValueDeserializer(bodyArgument, config);
         debugDeserializationConfiguration(method, config);
     }
 
     @SuppressWarnings({ "rawtypes", "unchecked" })
-    private void configureKeyDeserializer(Argument<?> bodyArgument, ExecutableMethod<?, ?> method, DefaultKafkaConsumerConfiguration config) {
+    private void configureKeyDeserializer(Optional<Argument<?>> bodyArgument, ExecutableMethod<?, ?> method, DefaultKafkaConsumerConfiguration config) {
```

Comment:
We generally try not to use `Optional` as a method parameter (though I know there are some cases in this project where that has been violated).

Can you refactor this to use the previous signature without `Optional`?